src/services/juxt-web/routes/console/userpage.ts

```diff
@@ -224,20 +262,10 @@ async function userRelations(req, res, userID) {
 		return res.redirect('/404');
 	}
 
-	let followers; let communities; let communityMap; let selection;
+	const communityMap = util.getCommunityHash();
 
 	if (req.params.type === 'yeahs') {
 		const posts = await POST.find({ yeahs: req.pid, removed: false }).sort({created_at: -1});
-		/*let posts = await POST.aggregate([
-            { $match: { id: { $in: likesArray } } },
-            {$addFields: {
-                    "__order": { $indexOfArray: [ likesArray, "$id" ] }
-                }},
-            { $sort: { "__order": 1 } },
-            { $project: { index: 0, _id: 0 } },
-            { $limit: config.post_limit }
-        ]);*/
```

Comment:
I'm not sure what you mean? The commented code? I removed all the commented out code I found in different files for a few reasons:
- It's committed there so can be found in git history if we want to go back to it
- It didn't have anything explaining why it should be kept
- A lot of the code wouldn't necessarily be correct after the TS rework anyway

In this specific example, no `likesArray` symbol is in scope, so the code doesn't really make sense in the context it's in. It looks like it's similar to the method below which I fully removed because it seemed not to be used anymore, and wouldn't have compiled